esmvaltool/diag_scripts/seaice/seaice_sensitivity.py

```diff
@@ -0,0 +1,343 @@
+        }
+        # Setting titles for plots
+        dictionary["titles"] = {
+            "notz_fig_title": "Annually Meaned Antarctic Sea Ice Sensitivity",
+            "notz_ax_title": "dSIA/dGMST",
+            "notz_plot_filename": "Annual Antarctic sea ice sensitivity",
+            "roach_fig_title": "Trends in Annual Mean Temperature And Annual Antarctic Sea Ice",
+            "roach_plot_filename": "Annual Antarctic sea ice trends",
+        }
+
+    return dictionary
+
+
+def notz_style_plot_from_dict(data_dictionary, titles_dictionary, cfg):
+    """Save a plot of sensitivities and observations."""
+    # Read from (Ed Blockley's) observations dictionary
+    obs_dict = titles_dictionary["obs"]
+    obs_years = list(obs_dict)[0]
+    obs_mean = obs_dict[obs_years]["mean"]
+    obs_std_dev = obs_dict[obs_years]["std_dev"]
+    obs_plausible = obs_dict[obs_years]["plausible"]
+
+    # Set up the figure
+    fig, ax = plt.subplots(figsize=(4, 6), layout="constrained")
+    fig.suptitle(titles_dictionary["titles"]["notz_fig_title"])
+    ax.set_title(titles_dictionary["titles"]["notz_ax_title"])  # Ed's title
+
+    # Iterate over the dictionary
+    for dataset, inner_dict in data_dictionary.items():
+        ax.plot(
+            0.5,
+            inner_dict["direct_sensitivity"],
+            label=dataset,
+            marker="_",
+            markersize=20,
+        )
+
+    # Add observations (style taken from Ed's code)
+    if obs_mean is not None:
+        ax.hlines(obs_mean, 0, 1, linestyle="--", color="black", linewidth=2)
+        ax.fill_between(
+            [0, 1],
+            obs_mean - obs_std_dev,
+            obs_mean + obs_std_dev,
+            facecolor="k",
+            alpha=0.15,
+        )
+        ax.hlines(
+            obs_mean + obs_plausible,
+            0,
+            1,
+            linestyle=":",
+            color="0.5",
+            linewidth=1,
+        )
+        ax.hlines(
+            obs_mean - obs_plausible,
+            0,
+            1,
+            linestyle=":",
+            color="0.5",
+            linewidth=1,
+        )
+
+    # Tidy the figure
+    ax.set_xticks([])
+    ax.set_ylabel("dSIA/dGMST (million km$^2$ K$^{-1}$)")
+    plt.legend(loc="center left", bbox_to_anchor=(1, 0.5))
+
+    # Save the figure (also closes it)
+    provenance_record = get_provenance_record(cfg)
+    save_figure(
+        titles_dictionary["titles"]["notz_plot_filename"],
+        provenance_record,
+        cfg,
+        figure=fig,
+        close=True,
+    )
+
+
+def roach_style_plot_from_dict(data_dictionary, titles_dictionary, cfg):
+    """Save a plot of trend in SIA against trend in GMST to the given filename."""
+    # Set up the figure
+    fig, ax = plt.subplots(figsize=(10, 6), layout="constrained")
+    fig.suptitle(titles_dictionary["titles"]["roach_fig_title"])
+    ax.set_title("Decades are scaled up from annual data")
```

Comment:
```suggestion
```
This could be stated in the documentation, but I don't think needs to be explained in the plot.